README.md

```diff
@@ -1,44 +1,60 @@
 [![Build Status](https://travis-ci.org/cloudfoundry-incubator/eirini.svg?branch=master)](https://travis-ci.org/cloudfoundry-incubator/eirini)
 
-# Eirini - what this?
+# What is Eirini?
 
-Eirini is a Kubernetes backend for Cloud Foundry. It deploys CF apps to a kube backend, using OCI images and Kube deployments.
+*Eirini* is a Kubernetes backend for Cloud Foundry.
+It deploys CF apps to a kube backend, using OCI images and Kube deployments.
 
-_But there's more!_
+Eirini exports staged CF images as docker images.
+So you can schedule them however you'd like.
+*And separately* it gives you the nice integrated `cf push` flow,
+with CF Apps mapped directly to kube deployment objects (pods).
+In other words it decouples buildpack staging and stateless-multitenant-app running.
 
-Eirini exports staged CF images as docker images. So you can schedule them however you'd like. *And separately* it gives you the nice integrated `cf push` flow, with CF Apps mapped directly to kube Deployment objects. In other words it decouples buildpack staging and stateless-multitenant-app running.
+Since scheduling is increasingly commoditized,
+Eirini provides an "Orchestrator Platform Interface (OPI)" layer,
+that abstracts away orchestration from Cloud Foundry's control plane.
+This means Eirini is not solely a Kube backend at all,
+but that it is a generic backend for any scheduler!
+This means it can schedule to diego, kube, swarm and other orchestration platforms,
```

Comment:
We'd prefer `could` over `can` because in reality, there only is Kubernetes.

Reply:
see above